test/test_prototype_transforms_kernels.py

```diff
@@ -92,7 +94,7 @@ class TestCommon:
     sample_inputs = pytest.mark.parametrize(
         ("info", "args_kwargs"),
         [
-            pytest.param(info, args_kwargs, id=f"{info.kernel.__name__}")
+            pytest.param(info, args_kwargs, id=f"{info.kernel_name}-")
```

Comment:
The extra `-` makes it more clear that the number is related to the sample input rather than to the kernel

`test_against_reference[resize_image_tensor-0]`

vs.


`test_against_reference[resize_image_tensor0]`